src/main/java/hudson/plugins/git/ChangelogToRevOptions.java

```diff
@@ -0,0 +1,39 @@
+package hudson.plugins.git;
+
+import java.io.Serializable;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import hudson.Extension;
+import hudson.model.AbstractDescribableImpl;
+import hudson.model.Descriptor;
+
+/**
+ * Options for the {@link hudson.plugins.git.extensions.impl.ChangelogToRev} extension.
+ *
+ * @author <a href="mailto:jacob.e.keller@intel.com">Jacob Keller (jacob.e.keller@intel.com)</a>
+ */
+public class ChangelogToRevOptions extends AbstractDescribableImpl<ChangelogToRevOptions> implements ChangelogOptions, Serializable {
+    private String revision;
+
+    @DataBoundConstructor
+    public ChangelogToRevOptions(String revision) {
+        this.revision = revision;
+    }
+
+    public ChangelogToRevOptions(ChangelogOptions options) {
+        this(options.getRevision());
+    }
+
+    public String getRevision() {
+        return revision;
+    }
+
+    @Extension
+    public static class DescriptorImpl extends Descriptor<ChangelogToRevOptions> {
+
+        @Override
+        public String getDisplayName() {
+            return "";
```

Comment:
Won't an empty string be a very difficult display name to identify?  Could you use a more descriptive string?

Reply:
This was copied from ChangeLogToBranch. Should they be different? (perhaps it should be the revision itself.

Reply:
Oh. I suppose it should actually be something, but not the empty string. I don't know what to put here, considering that the original ChangeLogToBranch code uses "".